playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -176,8 +176,12 @@
   - name: Verify OpenShift 3.2 RPMs are available for upgrade
     fail:
       msg: "OpenShift {{ avail_openshift_version.stdout }} is available, but 3.2 or greater is required"
-    when: not openshift.common.is_containerized | bool and not avail_openshift_version | skipped and avail_openshift_version.stdout | default('0.0', True) | version_compare('3.2', '<')
+    when: deployment_type != 'origin' and not openshift.common.is_containerized | bool and not avail_openshift_version | skipped and avail_openshift_version.stdout | default('0.0', True) | version_compare('3.2', '<')
```

Comment:
`version_compare(openshift_release,'<')` to cut down on the chances of updating one place to 3.3 in the future but not another?